### ISSUE TITLE
--net=host

### ISSUE THREAD

**Issue by Falven** [microsoft/azure-container-apps#563]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is a: (mark with an x)
- [X ] bug report -> please search issues before submitting
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)

### Issue description
Does the ACA Docker Runtime support --net=host?

**Command**
--net=host,--name=homebridge,-v,$(pwd)/homebridge:/homebridge

**Error:**
failed to create containerd task: failed to create shim task: OCI runtime create failed: container_linux.go:380: starting container process caused: exec: "--net=host": executable file not found in $PATH: unknown

### Steps to reproduce
1. Create an azure container Apps Instance
2. Set container to https://hub.docker.com/r/oznu/homebridge/
3. Set startup comamnd to `--net=host,--name=homebridge,-v,$(pwd)/homebridge:/homebridge`

**Expected behavior** [What you expected to happen.]
Container should start up sharing the host's network namespace. 

**Actual behavior** [What actually happened.]
Error

**Comment by ahmelsayed** [comment]
No, this is not possible. The `command` on the container is the [container's start command](https://learn.microsoft.com/en-us/rest/api/containerapps/container-apps/create-or-update?tabs=HTTP#container) not the docker's run command. i.e: the `COMMAND` in this [docker doc](https://docs.docker.com/engine/reference/commandline/run/) or this "Command" in the [kubernetes doc](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/)
